src/app/app-routing.module.ts

```diff
@@ -17,10 +17,13 @@ import {ChatComponent} from './chat/chat.component';
 import {ChatsComponent} from './chat/chats/chats';
 import {FeedbackComponent} from './feedback/feedback.component';
 import {VerificationComponent} from './verification/verification.component';
+import {MyProfileComponent} from "./my-profile/my-profile.component";
+import {AdminApproveStartComponent} from "./admin-approve-page/admin-approve-start/admin-approve-start.component";
+import {ServiceProviderViewComponent} from "./admin-approve-page/service-provider-view/service-provider-view.component";
+import {CustomerViewComponent} from "./admin-approve-page/customer-view/customer-view.component";
+import {FavoriteComponent} from "./favorite/favorite.component";
 import {PortfolioComponent} from './portfolio/portfolio.component';
 import {EditPostComponent} from './portfolio/edit-post/edit-post.component';
-import {AddPostComponent} from './portfolio/add-post/add-post.component';
-import {MyProfileComponent} from './my-profile/my-profile.component';
 import {NotFoundComponent} from './not-found/not-found.component';
```

Comment:
import {AddPostComponent} from './portfolio/add-post/add-post.component';  you delete this one - is it OK ?